x/liquidity/keeper/batch.go

```diff
@@ -35,7 +35,7 @@ func (k Keeper) DeleteAndInitPoolBatch(ctx sdk.Context) {
 			height := ctx.BlockHeight()
 
 			// reinitialize remaining batch msgs
-			// In the case of BatchSwapMsgs, it is often fractional matched or has not yet expired since it has not passed ExpiryHeight.
+			// In the case of BatchSwapMsgs, it is often fractionally matched or has not yet expired since it has not passed ExpiryHeight.
```

Comment:
❓ what do we mean by "it"
do we mean  "the batch messages are fractionally matched"? 
or is it a message in the batch that is fractionally matched? I'm not clear

Reply:
> ❓ what do we mean by "it"

It refers to the user's swap order.